aiida/backends/tests/restapi.py

```diff
@@ -789,6 +789,28 @@ def test_structure_visualization(self):
                                                         url,
                                                         response, uuid=node_uuid)
 
+    def test_xsf_visualization(self):
+        """
+        Get the list of given calculation inputs
+        """
+        from aiida.backends.tests.dataclasses import simplify
+        node_uuid = self.get_dummy_data()["structuredata"][0]["uuid"]
+        url = self.get_url_prefix() + '/structures/' + str(
+            node_uuid) + '/content/visualization'
```

Comment:
This is testing also if the default is XSF. 
I think it's better to test explicitly setting the xsf format tin the url.
Also, in general (in the future) probably better not to change the default or not even have one (and make sure already now, in any case, that the frontend always requires a specific format)